databricks/koalas/base.py

```diff
@@ -941,15 +943,23 @@ def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, d
         Name: koalas, dtype: int64
         """
         from databricks.koalas.series import Series, _col
+        from databricks.koalas.indexes import MultiIndex
+        if LooseVersion(pyspark.__version__) < LooseVersion("2.4") and \
+                default_session().conf.get("spark.sql.execution.arrow.enabled") == "true" and \
+                isinstance(self, MultiIndex):
+            raise RuntimeError("if you're using pyspark < 2.4, set conf "
+                               "'spark.sql.execution.arrow.enabled' to 'false' "
+                               "for using this function with MultiIndex")
```

Comment:
Btw, I think we should do this only in `MultiIndex` by overriding.

Reply:
ah, that seems better. thanks for the advice!